I.write-tests/06-largest-number/largest-number.test.js

```diff
@@ -2,4 +2,11 @@
 // input: [3, 21, 88, 4, 36];
 // expected: 88;
 
-// also test that the original array hasn't changed
\ No newline at end of file
+// also test that the original array hasn't changed
+const larg = require("./largest-number");
```

Comment:
Very small issue, but we tend to import functions with the same name, just so it's clear. So we would name this `largestNumber`.